R/utilities.R

```diff
@@ -842,11 +843,12 @@ maf_to_custom_track = function(maf_data,
 #' @import tidyverse config
 #'
 #' @examples
-#' everything_collated = collate_results(join_with_full_metadata = TRUE)
```

Comment:
Can we keep the old name of the argument to maintain backward compatibility?

Reply:
Totally!